pkg/credentials/s3/s3_secret.go

```diff
@@ -36,7 +36,7 @@ const (
 	S3Endpoint             = "S3_ENDPOINT"
 	S3UseHttps             = "S3_USE_HTTPS"
 	S3VerifySSL            = "S3_VERIFY_SSL"
-	S3UseVirtualBucket     = "S3_USER_VIRTUAL_BUCKET"
+	S3UseVirtualBucket     = "S3_USE_VIRTUAL_BUCKET"
```

Comment:
This has the risk of breaking backwards compatibility. @yuzisun Should I make this work for both and add a comment to remove the old one in major releases?

Reply:
Hey @terrytangyuan !

This key seems to be configurable via a ConfigMap.

IMO, it is going to be OK to fix the typo. We should, simply, state in release notes that users using the key with the typo can restore the old behavior by setting the desired key in the `inferenceservice-config` ConfigMap.